lib/db/util.js

```diff
@@ -4,10 +4,22 @@
 
 'use strict'
 
+const assert = require('assert')
 const crypto = require('crypto')
 const P = require('../promise')
 const randomBytes = P.promisify(require('crypto').randomBytes)
 const scryptHash = P.promisify(require('scrypt-hash'))
+const random = require('./random')
+
+const ALPHABET = '0123456789ABCDEFGHJKMNPQRSTVWXYZ'
+
+// some sanity checks, hard to test, private to this mdoule
+assert.equal(ALPHABET.length, 32, 'ALPHABET is 32 characters')
```

Comment:
Seeing this, makes me think it is better to put the `base32` function in the `/db/random.js` file. It will make things a little more clear. We can also create a new test file in `/tests/db/random` to test the function.